mdn/tests/test_tasks.py

```diff
@@ -99,11 +99,11 @@ def tearDown(self):
 
     def test_good_call(self):
         data = {
-            'locale': 'en-US', 'title': 'display',
-            'url': '/en-US/docs/Web/CSS/display',
+            'locale': 'en-US', 'title': 'float',
+            'url': '/en-US/docs/Web/CSS/float',
             'translations': [{
-                'locale': 'es', 'title': 'display',
-                'url': '/es/docs/Web/CSS/display'}]}
+                'locale': 'es', 'title': 'float',
+                'url': '/es/docs/Web/CSS/float'}]}
```

Comment:
Didn't see anything in the commit messages about changing from display to float as a good test call?